README.md

```diff
@@ -78,6 +78,39 @@ end
 See [documentation](https://hexdocs.pm/telemetry_poller/) for more concrete examples and usage
 instructions.
 
+## VM metrics example
+
+### Erlang
+
+Find, in `examples/telemetry_poller_vm.erl`, an example on how to retrieve to VM measurements,
+mentioned above.
+
+To see it in action, fire up `rebar3 shell`, then
+
+```erlang
+{ok, telemetry_poller_vm} = c("examples/telemetry_poller_vm").
+ok = file:delete("telemetry_poller_vm.beam").  % Deletes generated BEAM
+ok = telemetry_poller_vm:attach().
+```
+
+### Elixir
+
+Find, in `examples/TelemetryPollerVM.ex`, an example on how to retrieve to VM measurements,
+mentioned above.
+
+To see it in action, first compile the Erlang sources with `rebar3 compile`.
+
+Then fire up `iex`, then
```

Comment:
```suggestion
Then fire up `iex -S mix`, then
```

Then you don't need to either start nor add code paths?

Reply:
Oh, ignore me, this is an Erlang project.

Reply:
It was either this or adding a `mix.exs`. I could also do that, but then it would be confusing. Another alternative I took into account would be to have a fully working example (rebar and mix) inside the `examples`, but it felt like overkill :)